modular_skyrat/modules/New_drinks/code/modules/food_and_drinks/recipes/alcohol_reagents.dm

```diff
@@ -0,0 +1,23 @@
+/datum/reagent/consumable/ethanol/cum_in_a_hot_tub
+	name = "Cream in the Hot Tub"
+	boozepwr = 80
+	color = "#D2D7D9"
+	quality = DRINK_VERYGOOD
+	taste_description = "smooth cream"
+	glass_icon = 'modular_skyrat/modules/New_drinks/icons/obj/CIAH.dmi'
+	glass_icon_state = "cum_glass"
+	shot_glass_icon_state = "cum_shot"	//I'm funny, I know
+	glass_name = "Cum in the Hot Tub"
```

Comment:
```suggestion
	glass_name = "Cream in the Hot Tub"
```
If I'm understanding this correctly, this hasn't actually changed the name the glass shows, just the name of the actual reagent, which doesn't actually address the issue of prefbreak

Reply:
though personally the unique description is enough in my eyes, for this. Or at least, make the second one more obvious that it's.. you know.

Reply:
To be honest I'd rather the name accurately depicts what, besides the hot tub part, it is.
Some big baby is going to whine and make a fuss trying to get someone banned because they don't know what the drink is and drunk naughty reagent. I don't know what drinks are either, but I'm not going to ahelp prefbreak if I unwittingly drink yellow snow or something, but some people are.

Reply:
Nvm you're right, this reagent doesn't even have the forbidden (by prudes) reagent in it.